Makefile

```diff
@@ -21,3 +22,4 @@ include $(addprefix ./vendor/github.com/openshift/library-go/alpha-build-machine
 # It will generate target "image-$(1)" for builing the image an binding it as a prerequisite to target "images".
 $(call build-image,origin-node,./images/sdn/Dockerfile,.)
 $(call build-image,origin-sdn-controller,./images/sdn-controller/Dockerfile,.)
+$(call build-image,origin-kube-proxy,./images/kube-proxy/Dockerfile,.)
```

Comment:
I am not able to test that lol, what exact make cmd would I call there? 
`make build-image`? nope
`make kube-proxy`? nope
`make all`? nope
I need help reading that Makefile i guess :laughing:

Reply:
`make -p blah` will try and fail to make `blah` and then output the post-processed form of the Makefile, which you can then dig through to find that the magic words are `make image-origin-kube-proxy`

Reply:
thanks! :)